pkg/rpc/response/result/application_log.go

```diff
@@ -10,7 +10,7 @@ import (
 // ApplicationLog wrapper used for the representation of the
 // state.AppExecResult based on the specific tx on the RPC Server.
 type ApplicationLog struct {
-	TxHash      util.Uint256              `json:"txid"`
+	TxHash      *util.Uint256             `json:"txid"`
```

Comment:
It's not correct, this field should always be present and in case of block this should be a block hash. We'll discuss this extension with NGD later, but for now that's the way.

Reply:
Ok